staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -167,6 +168,7 @@ func NewCmdLogs(f cmdutil.Factory, streams genericiooptions.IOStreams) *cobra.Co
 }
 
 func (o *LogsOptions) AddFlags(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(&o.AllPods, "all-pods", o.AllPods, "Get logs from all pod(s). Sets prefix to true.")
 	cmd.Flags().BoolVar(&o.AllContainers, "all-containers", o.AllContainers, "Get all containers' logs in the pod(s).")
```

Comment:
parameter named "allContainers" in the function "NewLogsOptions" has be removed.

Reply:
I am not sure I understand?

The flag `--all-containers` was not removed, just the parameter from the constructor.